samples/emoji-search/README.md

```diff
@@ -7,12 +7,16 @@ This is a mobile app demo of Zipline.
  * **presenter-treehouse** is a Kotlin/Multiplatform library which exposes the presenter to a host application.
  * **android** is an Android application that downloads the presenter JavaScript and displays it.
  * **ios** is an iOS application that downloads the presenter JavaScript and displays it.
+ * **browser** is a web application that downloads the presenter JavaScript and displays it.
+
 
 Prerequisites
 -------------
 
-In order to build and run these applications you'll need to:
-- Have Android Studio installed
+In order to build and run these applications you'll need to have the following installed:
+- Android Studio
+- gradle
+- git-lfs
```

Comment:
What do we need git-lfs for?

Reply:
Honestly I'm not sure, but when I did not have git-lfs installed, `./gradlew samples:counter:browser:browserRun` failed to complete and threw some error that referred to git-lfs. Once I installed it, everything worked.

Reply:
Despite being a member of treehouse-maintainers, I don't have write access to this repository, so someone will need to merge this for me.

Reply:
The paparazzi snapshots are stored in LFS